amplitude.py

```diff
@@ -1,20 +1,35 @@
+import hashlib
+import hmac
 import json
+import os
+import requests
 import sys
 
+API_KEY = os.environ["FXA_AMPLITUDE_API_KEY"]
+HMAC_KEY = os.environ["FXA_AMPLITUDE_HMAC_KEY"]
+
 def transform(line):
     try:
-        parsed_line = json.loads(line)
-
-        assert(type(parsed_line) is dict)
-
-        event = parsed_line["Fields"]
+        event = json.loads(line)["Fields"]
 
         assert("time" in event)
         assert("user_id" in event or "device_id" in event)
 
+        if "user_id" in event:
+            assert(event["user_id"] == event["user_properties"]["fxa_uid"])
+
+            user_id_hmac = hmac.new(HMAC_KEY, event["user_id"], hashlib.sha256)
+            event["user_id"] = user_id_hmac.hexdigest()
+            event["user_properties"]["fxa_uid"] = event["user_id"]
```

Comment:
If we're doing this here, it's redundant to set `user_properties\.fxa_uid` in our logging code in the auth and content servers. We should remove that if/when this sticks.

Reply:
(and get rid of the assertion on line 19 too, of course)

Reply:
> ...it's redundant to set `user_properties.fxa_uid`...

@davismtl raised this independently, we can get rid of it entirely.